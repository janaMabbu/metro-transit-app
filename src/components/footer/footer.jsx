import React from 'react'
import './footer.less'

export const Footer = () => {
  return (
      <footer className="col-sm-12 app-footer__content">
      <p className="app-footer__sub-content" >Metro Transit is a service of the Metropolitan Council.</p>
      <p className="app-footer__sub-content1">Minneapolis/St. Paul, MN</p>
      <p className="app-footer__copy-right">Â© 2019 Metro Transit</p>
      </footer>
    )
}