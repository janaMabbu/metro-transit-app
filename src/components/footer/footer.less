@import "~less/common.less";

.app-footer {
  &__content {
    color: @lightest-grey;
    background: @white-human;
    font-size: 14px;
    text-align: center;
    padding: 20px;
    box-sizing: border-box;
  }

  &__sub-content,&__sub-conten1,&__copy-right {
   margin-bottom:5px;
  }

}