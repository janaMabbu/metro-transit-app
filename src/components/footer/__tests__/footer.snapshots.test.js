import { shallow } from 'enzyme'
import React from 'react'

import { Footer } from 'components/footer/footer'

describe('Component - Footer', () => {
  it('should render the Footer snapshot', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })

  it('it should contain content and copy-right', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper.find('.app-footer__content').length).toBe(1)
    expect(wrapper.find('.app-footer__sub-content').length).toBe(1)
    expect(wrapper.find('.app-footer__sub-content1').length).toBe(1)
    expect(wrapper.find('.app-footer__copy-right').length).toBe(1)
  })

})