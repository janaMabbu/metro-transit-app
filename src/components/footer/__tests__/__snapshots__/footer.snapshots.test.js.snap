// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Footer should render the Footer snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="app-footer__sub-content"
        >
          Metro Transit is a service of the Metropolitan Council.
        </p>,
        <p
          className="app-footer__sub-content1"
        >
          Minneapolis/St. Paul, MN
        </p>,
        <p
          className="app-footer__copy-right"
        >
          © 2019 Metro Transit
        </p>,
      ],
      "className": "col-sm-12 app-footer__content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Metro Transit is a service of the Metropolitan Council.",
          "className": "app-footer__sub-content",
        },
        "ref": null,
        "rendered": "Metro Transit is a service of the Metropolitan Council.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Minneapolis/St. Paul, MN",
          "className": "app-footer__sub-content1",
        },
        "ref": null,
        "rendered": "Minneapolis/St. Paul, MN",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "© 2019 Metro Transit",
          "className": "app-footer__copy-right",
        },
        "ref": null,
        "rendered": "© 2019 Metro Transit",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="app-footer__sub-content"
          >
            Metro Transit is a service of the Metropolitan Council.
          </p>,
          <p
            className="app-footer__sub-content1"
          >
            Minneapolis/St. Paul, MN
          </p>,
          <p
            className="app-footer__copy-right"
          >
            © 2019 Metro Transit
          </p>,
        ],
        "className": "col-sm-12 app-footer__content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Metro Transit is a service of the Metropolitan Council.",
            "className": "app-footer__sub-content",
          },
          "ref": null,
          "rendered": "Metro Transit is a service of the Metropolitan Council.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Minneapolis/St. Paul, MN",
            "className": "app-footer__sub-content1",
          },
          "ref": null,
          "rendered": "Minneapolis/St. Paul, MN",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "© 2019 Metro Transit",
            "className": "app-footer__copy-right",
          },
          "ref": null,
          "rendered": "© 2019 Metro Transit",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
